src/Model/Command/UserRecommendation.php

```diff
@@ -204,6 +242,10 @@ protected function getCommandParameters(): array
             $parameters['model_name'] = $this->modelName;
         }
 
+        if ($this->responseProperties !== null) {
```

Comment:
So you don't send item_id, and it is always returned. In this case I would omit item_id from the examples above and just explain it is always included, and you define just additional properties.